docs/content/querying/groupbyquery.md

```diff
@@ -214,3 +215,4 @@ When using the "v2" strategy, the following query context parameters apply:
 |`maxMergingDictionarySize`|Can be used to lower the value of `druid.query.groupBy.maxMergingDictionarySize` for this query.|
 |`maxOnDiskStorage`|Can be used to lower the value of `druid.query.groupBy.maxOnDiskStorage` for this query.|
 |`sortByDimsFirst`|Sort the results first by dimension values and then by timestamp.|
+|`tmpDir`|Overrides the value of `druid.query.groupBy.tmpDir` for this query.|
```

Comment:
I'd leave this out. I can't think of a good reason for letting a user override the temp dir used on Druid servers, and this is a security hole (user could set tmpDir = somewhere the user shouldn't be able to scribble on).